packages/create-schemas/src/utils.ts

```diff
@@ -67,3 +67,23 @@ export function toFullyQualifiedURL(
 
     return pathToFileURL(join(process.cwd(), root, input)).toString();
 }
+
+export function formatBytes(bytes: number) {
```

Comment:
wonder if there is a library that can help with that?

Reply:
Probably. 

When things are simple like that, I usually prefer reimplementing the function rather than importing it from a library, but I know different people have different opinions on that.